app/demo-loop/Transporter.cc

```diff
@@ -215,96 +165,29 @@ TransporterResult Transporter<M>::operator()(const TrackInitParams& primaries)
         result.active.reserve(input_.max_steps);
```

Comment:
Yep, can do. Since there's no extra overhead for collecting the alive/active/queued statistics I think it's OK to leave them on. The only downside is the extra JSON output, which we can filter without too much trouble.

Reply:
Yeah the extra JSON output gets pretty big for large runs, but you're right, easy to filter.